### PR TITLE
[B] Fix a11y issues

### PR DESCRIPTION
**Pull request by jen-castiron**
- Add missing alt text in thumbnail images
- Add missing delete button text
- Add missing lang attribute

### CODE REVIEW

File: components/atomic/buttons/ButtonControl/patterns/ButtonControlConfirm.tsx [hunk 1]
```diff
@@ -38,15 +36,16 @@ const ButtonControlConfirm = forwardRef(
 
     return (
       <>
-        <Styled.Disclosure
+        <DialogDisclosure
+          as={ButtonControl}
```
Comment (jen-castiron) [inline components/atomic/buttons/ButtonControl/patterns/ButtonControlConfirm.tsx:40]: @1aurend Something I totally forgot about Reakit components - they can be styled by setting `as` prop to a styled component! I updated the delete button to use this pattern, so we don't have to repeat styles. ✨ I didn't see any mobile issues, but let me know if something accidentally regressed.

Comment (1aurend) [reply components/atomic/buttons/ButtonControl/patterns/ButtonControlConfirm.tsx:40]: Nice @jen-castiron! It looks great to me!

Comment (1aurend) [reply components/atomic/buttons/ButtonControl/patterns/ButtonControlConfirm.tsx:40]: Follow up— I'm working on implementing the `ButtonControlGroup` that accepts children and I've come across two related things:
- `ButtonControlDrawer` appears not to be tabbable?
- the styles in `ButtonControlDrawer` need to be modified in parallel to what you've done here for `ButtonControlConfirm` so the icons line up in a dropdown:

Screen Shot 2021-10-08 at 10 25 12 AM

At this point, the only place we'd be using `ButtonControlDrawer` in a dropdown is storybook so it's not urgent.

Comment (1aurend) [reply components/atomic/buttons/ButtonControl/patterns/ButtonControlConfirm.tsx:40]: I'm using Firefox

**Pull request merged**